src/libs/dps-cache-starter/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/cache/RedisStorageService.java

```diff
@@ -62,5 +62,19 @@ public String put(byte[] content) {
 
     }
 
+    /**
+     * Removes a document from redis key value store.
+     * @param key    object key to evict from storage
+     */
+    @Override
+    public void delete(String key) {
```

Comment:
Should we return if the key is null or empty?

Reply:
Done!